MC-35509: Updated the software update guide module with the safe upgrade tool documentation (#8314)
magento/devdocs: src/guides/v2.3/comp-mgr/bk-compman-upgrade-guide.md

Complete the tasks discussed in [Prerequisites].

 {:.bs-callout-info}

> See the [Safe upgrade tool section]({{ site.baseurl }}/safe-upgrade-tool/introduction.html) for more information on the new CLI magento tool that helps you update your magento software.

Reviewer: ```suggestion
See the [Safe Upgrade Tool ]({{ site.baseurl }}/safe-upgrade-tool/introduction.html) for more information about the new Magento CLI tool that helps you update your Magento software.
```

Reviewer 2: Could we add "(ALPHA)" next to Safe Upgrade Tool ?

Author: added
